packages/tailwind-config/index.js

```diff
@@ -131,8 +131,16 @@ export default {
         },
       },
       backgroundImage: {
-        'card-radial':
-          'radial-gradient(33% 50% at 15% 44%, var(--rust), transparent),radial-gradient(33% 40% at 105% 42%, var(--teal), transparent),radial-gradient(33% 80% at 85% 124%, var(--teal), transparent)',
+        // The final `linear-gradient` is just to make a solid charcoal
+        // background color for the radial gradients to sit on top of. If
+        // there's a way to make a solid background color without
+        // `linear-gradient`, feel free to update this.
+        'card-radial': `
```

Comment:
Three changes here:
1. I formatted this string with line breaks to make it more readable.
2. I added `color-mix()` to set the radial gradients to 20% opacity. Previously, this was done by making the div on which the gradients appeared 20% opaque.
3. I added a `linear-gradient` at the bottom to form the background color.